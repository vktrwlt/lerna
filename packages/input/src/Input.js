import styled from 'styled-components';

export const Input = styled.input`
    border-radius: 8px;
    border: 2px solid #0f6;

    font-size: 16px;
    font-family: Verdana, Geneva, Tahoma, sans-serif;

    padding: 16px 16px;

    width: 200px;

    &:active{
        outline: none;
    }

    &:focus{
        outline: none;
    }
`