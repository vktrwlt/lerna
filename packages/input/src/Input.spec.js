import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { Input } from "./Input";

describe("Input", () => {
  test("render component", () => {
    const tree = renderer.create(<Input placeholder="Name" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
