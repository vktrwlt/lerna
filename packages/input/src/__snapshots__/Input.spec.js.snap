// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Renderizou corretamente 1`] = `
<input
  className=""
  placeholder="Nome"
/>
`;
